dietpi/dietpi-software

```diff
@@ -304,8 +304,7 @@ _EOF_
 		aSOFTWARE_DESC[$software_id]='ultra lightweight desktop'
 		aSOFTWARE_CATEGORY_INDEX[$software_id]=0
 		aSOFTWARE_ONLINEDOC_URL[$software_id]='https://dietpi.com/docs/software/desktop/#lxde'
-		aSOFTWARE_REQUIRES_ALSA[$software_id]=1
-		aSOFTWARE_REQUIRES_XSERVERXORG[$software_id]=1
+		aSOFTWARE_REQUIRES=(5 6)
```

Comment:
We need to make it an array: `aSOFTWARE_REQUIRES[$software_id]`
Since nested arrays are not possible in bash, the dependencies need to be a simple string:
```
aSOFTWARE_REQUIRES[$software_id]='5 6'
```

The loop hence needs to be:
```
for j in ${aSOFTWARE_REQUIRES[$1]}
```
without quotes. Shellcheck will complain, but we must ignore it (# shellcheck disable=SCXXXX). At least currently I have no better idea how to handle it without making it much more complicated.